### ISSUE TITLE
Regression

### ISSUE THREAD

**Issue by karasjoh000** [ralfbiedert/ffsvm-rust#4]
Just wondering if regression is supported e-SVM and v-SVM

**Comment by karasjoh000** [comment]
```
In particular:

svm_type must be c_svc.
kernel_type must be rbf or linear
All support vectors (past the SV line) must have strictly increasing attribute identifiers, without skipping an attribute.
```

**Comment by karasjoh000** [comment]
If there is, can you show an example load a regression problem? 
The Modelfile::try_from does not seem to work

**Comment by ralfbiedert** [comment]
Hi, yes, regression should work. That part of the documentation is outdated. I'll push 0.7 later today which fixes that and #5.
